src/app/components/url-box/url-box.component.ts

```diff
@@ -9,12 +10,13 @@ export class UrlBoxComponent {
   @Input() httpVerb: string;
   @Input() isSubscribed = false;
   @Input() isLoading = false;
+  @Input() isMac = navigator.appVersion.includes('Mac');
```

Comment:
You don't pass this variable to the component, so you don't need the `@Input()` in front of it.